### PR TITLE
Adds pytest-mpl

### PR DESCRIPTION
**Pull request by palnabarun**
Please check whether the way I am invoking the test is okay. I am asking this as I changed the approach a bit and I don't know confidently the differences between them.

### CODE REVIEW

**Comment by tacaswell** [comment]
I still find the coverage tools a bit magical, but I think this incantation is closer to what matplotlib uses so I suspect it will work.

Can you add a hello-world test of just creating a figure?

**Comment by palnabarun** [comment]
Tests are working fine locally. It is breaking on Travis CI.
I feel it is something to do with the environment.

**Comment by tacaswell** [comment]
I was wrong about what pytest-mpl did, it only affects the backend when you are doing image comparison tests, not globally (which on consideration is the better behavior).

I pushed a commit setting the backend env to agg which will hopefully fix this.

File: mplaltair/tests/test_basic.py [hunk 1]
```diff
@@ -1,3 +1,10 @@
+import matplotlib.pyplot as plt
+
 def test_import():
     import mplaltair
     del mplaltair
+
+def test_mpl_figure():
+    fig = plt.figure()
```
Comment (story645) [inline mplaltair/tests/test_basic.py]: I think mpl test may have cleanup code or maybe plt.close()

File: mplaltair/tests/test_basic.py [hunk 2]
```diff
@@ -1,3 +1,10 @@
+import matplotlib.pyplot as plt
+
 def test_import():
     import mplaltair
     del mplaltair
```
Comment (story645) [inline mplaltair/tests/test_basic.py]: Does this need the delete?

**Pull request merged**